src/Bridges/Constraint/norm_to_lp.jl

```diff
@@ -0,0 +1,52 @@
+"""
+    NormOneBridge{T}
+
+The `NormOneCone` is representable with LP constraints, since ``t \\ge \\sum_i |x_i|`` if and only if there exists a vector y such that ``t = \\sum_i y_i`` and ``y_i \\ge x_i`` and ``y_i \\ge -x_i`` for all ``i``.
+"""
+struct NormOneBridge{T} <: AbstractBridge
+    y::Vector{MOI.VariableIndex}
+    eq_index::CI{MOI.ScalarAffineFunction{T}, MOI.EqualTo{T}}
+    ge_index::CI{MOI.VectorAffineFunction{T}, MOI.Nonnegatives}
+end
+function bridge_constraint(::Type{NormOneBridge{T}}, model, f::MOI.VectorOfVariables, s::MOI.NormOneCone) where T
+    return bridge_constraint(NormOneBridge{T}, model, MOI.VectorAffineFunction{T}(f), s)
+end
+function bridge_constraint(::Type{NormOneBridge{T}}, model, f::MOI.VectorAffineFunction{T}, s::MOI.NormOneCone) where T
+    f_scalars = MOIU.eachscalar(f)
+    t = f_scalars[1]
+    x = f_scalars[2:end]
+    d = MOI.dimension(s) - 1
+    y = MOI.add_variables(model, d)
+    eq_index = MOI.add_constraint(model, t - sum(y), MOI.EqualTo(zero(T))) # TODO fix
```

Comment:
@blegat what is the best way of doing the `t - sum(y)` here?

Reply:
Currently `sum` will be inefficient as it won't exploit the mutability of the temporary affine expressions created (cc @Peiffap ). You an build the sum manually with `MOI.ScalarAffineFunction(...)` or use a `for` loop and `operate!(+, ...)`

Reply:
If sum will be faster in future then wouldn't it make more sense to use sum now, rather than the more complicated way?